### PR TITLE
Setting for reading time

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | N/A
| License       | MIT

Add a setting in config screen to define the reading speed for each user (I read so fast, I read so slowly, etc.)
TODO : 
- [x] tests
- [x] translations
- [x] documentation
- [x] baggy view
- [x] add label (instead of the numeric value on config screen)

### CODE REVIEW

**Comment by tcitworld** [comment]
Relaunched the Travis task that failed to clear cache for some reason.

File: src/Wallabag/CoreBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -29,6 +29,9 @@ public function getConfigTreeBuilder()
                 ->integerNode('rss_limit')
                     ->defaultValue(50)
                 ->end()
+                ->integerNode('reading_speed')
+                    ->defaultValue(200)
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/DependencyInjection/Configuration.php]: Why 200 ?

Comment (nicosomb) [inline src/Wallabag/CoreBundle/DependencyInjection/Configuration.php]: Will fix. I didn't fix some tests.

**Comment by nicosomb** [comment]
I removed the slider to put a list. More readable. 
I also added this text, below the list: 

> You can use online tools to estimate your reading speed ([like this one](http://www.myreadspeed.com/calculate/)).


capture du 2016-03-16 12 53 51

File: docs/en/user/configuration.rst [hunk 2]
```diff
@@ -21,6 +21,12 @@ Items per page
 
 You can change the number of articles displayed on each page.
 
+Reading speed
+~~~~~~~~~~~~~
+
+wallabag calculates a reading time for each article. You can define here, thanks to the slider, if you are
```
Comment (j0k3r) [inline docs/en/user/configuration.rst]: This is not a slider anymore

File: src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig [hunk 3]
```diff
@@ -26,6 +26,15 @@
 
         <fieldset class="w500p inline">
             <div class="row">
+                {{ form_label(form.config.reading_speed) }}
+                {{ form_errors(form.config.reading_speed) }}
+                {{ form_widget(form.config.reading_speed) }}
+            </div>
+        </fieldset>
+
+
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig]: Extra lines not needed

File: src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig [hunk 4]
```diff
@@ -26,6 +26,15 @@
 
         <fieldset class="w500p inline">
             <div class="row">
+                {{ form_label(form.config.reading_speed) }}
+                {{ form_errors(form.config.reading_speed) }}
+                {{ form_widget(form.config.reading_speed) }}
+            </div>
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig]: Baggy doesn't have the help message (about calculating reading time)?

**Comment by nicosomb** [comment]
Just improved tests ... 
I don't know if we have to add a test for this line https://github.com/wallabag/wallabag/pull/1767/files#diff-7422bc94ce6c3b3e5fd993868d036c88R53

**Comment by j0k3r** [comment]
I'm ok with that PR.
We just need to remember that if one day we are chaging the 200 values from `Tools/Utils.php` every _not default_ reading speed will be wrong.

File: src/Wallabag/CoreBundle/Entity/Config.php [hunk 5]
```diff
@@ -248,6 +255,30 @@ public function getRssLimit()
     }
 
     /**
+     * Set readingSpeed.
+     *
+     * @param int $readingSpeed
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Entity/Config.php]: This should be `float` right? (same for the `getReadingSpeed`)

**Pull request merged**